swarm.go

```diff
@@ -26,6 +26,11 @@ import (
 // protocol selection as well the handshake, if applicable.
 var DialTimeout = 60 * time.Second
 
+// DialTimeoutLocal is the maximum duration a Dial to local network address
+// is allowed to take. This only includes establishing the raw network
```

Comment:
I'm pretty sure this dial timeout will also include negotiating the security protocol etc.

Reply:
oh, right